### ISSUE TITLE
Ability to permanently close / disable specific tabs.

### ISSUE THREAD

**Issue by GainingAccess** [rabrain/ai-chat#19]
Hello. I use this program and I like it a lot. However, I was wondering if it was possible to disable some tabs that I don't use. Specifically, although I use the ChatGPT, Claude, and Gemini Tabs, I don't use the Copilot, Perplexity, Pi, or Poe tabs and I am wondering if there is some way to disable that.

**Comment by lilac** [comment]
Yes, it's in our internal roadmap. We would add a preference window, allowing user to customize the app features. But we're a small team, so please be patient on this. Thanks for your feedback.

**Comment by GainingAccess** [comment]
Okay, thanks 👍

**Comment by alsyundawy** [comment]
+1
